src/SFA.DAS.Campaign.Web/ViewComponents/TrainingProvider-Heading/TrainingProviderHeadingViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Sfa.Das.Sas.ApplicationServices.Models;
+using Sfa.Das.Sas.Shared.Components.ViewComponents.Fat;
+using Sfa.Das.Sas.Shared.Components.ViewComponents.TrainingProvider.Search;
+using Sfa.Das.Sas.Shared.Components.ViewModels;
+
+namespace SFA.DAS.Campaign.Web.ViewComponents
+{
+    public class TrainingProviderHeadingViewModel
```

Comment:
This model seems overkill for whats displayed on the page.

Reply:
it could be simplified to just the properties which are used in the heading currently but would mean any changes would require additional logic. My thinking was the information in headings within the campaign has changed pretty frequently, by doing this it means all information is available to the view without additional changes. is there a benefit in simplifying the model?

Reply:
I'd say clarity of the view and what it expects. In its naming it is the model of the view. I think it can confuse things by passing much more data. Mapping tests are also simpler.